runtime/lua/vim/treesitter/dev.lua

```diff
@@ -76,7 +76,9 @@ end
 ---
 ---@package
 function TSTreeView:new(bufnr, lang)
-  local parser = vim.treesitter.get_parser(bufnr or 0, lang, { error = false })
+  bufnr = bufnr or 0
+  lang = lang and lang or vim.treesitter.language.get_lang(vim.bo[bufnr].filetype)
```

Comment:
Maybe I forgot how Lua boolean logic works but can't this simply be:
```suggestion
  lang = lang or vim.treesitter.language.get_lang(vim.bo[bufnr].filetype)
```